catalogue/webapp/pages/images.tsx

```diff
@@ -53,6 +53,7 @@ type ImagesPaginationProps = {
   results: CatalogueResultsList<Image>;
   imagesRouteProps: ImagesRouteProps;
   setSavedSearchState: (state: ImagesRouteProps) => void;
+  hideMobilePaging?: boolean;
```

Comment:
could we call this `hideMobilePagination` for consistency?

Reply:
yep sounds good